nixos/modules/services/misc/paperless.nix

```diff
@@ -335,8 +341,6 @@ in
       environment = env;
 
       preStart = ''
-        ln -sf ${manage} ${cfg.dataDir}/paperless-manage
```

Comment:
How about keeping the symlink to ` ${cfg.dataDir}/paperless-manage`?
Without this, user scripts can only access `manage` via the global PATH environment.
Before symlinking, the paperless module had a read-only option `manage`.

Reply:
If we decide to remove this, it should probably be deleted too

Reply:
> How about keeping the symlink to ` ${cfg.dataDir}/paperless-manage`?

It is pretty none standard to have a random management tool in the /var/lib directory.

> Without this, user scripts and services can only access manage via the global PATH environment.

or via ``/run/current-system/sw/bin/paperless-manage directly``

> Before symlinking, the paperless module had a read-only option manage.

I don't think we necessarily need this.